flex-model/styles/components-style/about-us.css

```diff
@@ -0,0 +1,121 @@
+/* About Us */
+.about-us {
+  display: flex;
+  padding: 61px 15px;
+}
+
+.specialist {
+  display: flex;
+  max-width: 740px;
+  min-height: 196px; 
+  margin: 0 15px 58px;
+}
+
+.specialist:last-child {
+  margin-bottom: 0;
+}
+
+.specialist-photo {
+  max-width: 190px;
+  max-height: 195px;
+  border-radius: 6px;
+  user-select: none;
+}
+
+.specialists-article {  
+  margin-left: 26px;
+  font-family: 'Source Sans Pro';
```

Comment:
Here you need to add more fonts.
You need to add a system font at the   ##end.
_For example:_
`font-family: 'Source Sans Pro', sans-serif;
`